### DISCUSSION TITLE
Issue with PyPI installation

### DISCUSSION THREAD

**Discussion by AEnyChris** [apache/airflow#34229] [Q&A]
Unable to install aiflow using the PyPI command with constraint on Ubuntu 18.04 LTS. command bellow: 
`pip install 'apache-airflow==2.7.1'  --constraint "https://raw.githubusercontent.com/apache/airflow/constraints-2.7.1/constraints-3.8.txt"`

Getting error:
'ReadTimeoutError("HTTPSConnectionPool(host='raw.githubusercontent.com', port=443): Read timed out. (read timeout=15)",)': /apache/airflow/constraints-2.7.1/constraints-3.8.txt

**Comment by Taragolis** [comment]
Common issue, not related to Airflow itself.

It is either a problem with connection on your side or something wrong (rare case) with PyPI repository or something bad with Github (happen time to times)
- Check internet connection
- Check VPN connection
- If you use a proxy, then... check it
- In addition you could increase timeouts, see PIP docs https://pip.pypa.io/en/stable/topics/configuration/
